google-cloud-bigtable/pom.xml

```diff
@@ -323,11 +323,8 @@
                     <bigtable.data-endpoint>${bigtable.directpath-data-endpoint}</bigtable.data-endpoint>
                     <bigtable.admin-endpoint>${bigtable.directpath-admin-endpoint}</bigtable.admin-endpoint>
                     <bigtable.grpc-log-dir>${project.build.directory}/test-grpc-logs/directpath-it</bigtable.grpc-log-dir>
+                    <bigtable.attempt-directpath>true</bigtable.attempt-directpath>
                   </systemPropertyVariables>
-                  <!-- Enable directpath for bigtable -->
-                  <environmentVariables>
-                    <GOOGLE_CLOUD_ENABLE_DIRECT_PATH>bigtable</GOOGLE_CLOUD_ENABLE_DIRECT_PATH>
```

Comment:
Make sure to update AbstractTestEnv as well:
https://github.com/googleapis/java-bigtable/blob/7b7b2f26f0db35c4697ca49c8631f3528c2aec42/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/AbstractTestEnv.java#L78

Reply:
good catch, thx

Reply:
just for my understanding, how come we are switching from env variable to system property?

Reply:
We have decided to use [setAttemptDirectPath](https://github.com/googleapis/gax-java/blob/master/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java#L582) instead of the env var, for client library to enable DirectPath. The system property is temporarily used for testing purposes. Once DirectPath ipv4 support is fully rolled out, we will need to have a new release of the client library, with this sys property being removed, and setAttemptDirectPath being TRUE by default.